src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps;
+ 
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.List;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+ 
+public final class QuizTimingPropertiesUtils {
+ 
+    //This function gets the the "quiz_timestamp" property of the entity that is fed into the function
+    public Object getTimestampProperty(String entity, DatastoreService datastore) {
+        try {
+            Query query = new Query(entity);
+            PreparedQuery pq = datastore.prepare(query);
```

Comment:
Typically, we only want to have one operation in a try catch block. In this case, making a new query object likely won't result in an error so we can put that outside the try block and only have the PreparedQuery in the try block.